app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java

```diff
@@ -7,6 +7,7 @@
 import android.os.Bundle;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.Log;
```

Comment:
please remove unused imports "Log" and "CalendarFragment"

Reply:
done.